python/cuml/test/test_random_forest.py

```diff
@@ -69,7 +69,7 @@ def test_rf_classification(datatype, split_algo,
     # Initialize, fit and predict using cuML's
     # random forest classification model
     cuml_model = curfc(max_features=1.0,
-                       n_bins=8, split_algo=split_algo, split_criterion=0,
+                       n_bins=16, split_algo=split_algo, split_criterion=0,
```

Comment:
out of curiosity, why the n_bins change?

Reply:
Maybe n_bins=16 should be our default then? Want to make sure people have a good accuracy experience out of the box. (ok to defer decision post-commit though)